### DISCUSSION TITLE
Why the action is not running?

### DISCUSSION THREAD

**Discussion by JiahaoYao** [community/community#24322] [Apps, API and Webhooks]
<p>i created the workflow here, but there is no one running?</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/JiahaoYao/mytest/actions" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/20907377?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/JiahaoYao/mytest/actions" target="_blank" rel="noopener nofollow ugc">JiahaoYao/mytest</a></h3>

<p><a href="https://github.com/JiahaoYao/mytest/actions" target="_blank" rel="noopener nofollow ugc">ns</a></p>

  <p><span class="label1">Contribute to JiahaoYao/mytest development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>can anyone know the reason? Or it takes time.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The one with a <code>schedule</code> trigger (<code>stale.yml</code>) is set to run at 1:30 UTC, which was about 6.5 h ago. So you’ll have to wait about 17.5 h, or change the <code>cron</code> setting to a time that’s more convenient for you.</p>
<p>The others run on releases, issues, or pull requests, and I don’t see any of those in your repository.</p>

**Comment by JiahaoYao** [comment]
<p>Thanks <a class="mention" href="/airtower-luna">@airtower-luna</a>, appreciate your reply!</p>
